build.sbt

```diff
@@ -3,13 +3,39 @@ import uk.gov.hmrc.sbtdistributables.SbtDistributablesPlugin.publishingSettings
 
 val appName = "income-tax-employment"
 
-val silencerVersion = "1.7.3"
+val silencerVersion = "1.7.0"
```

Comment:
could this be 1.7.3?

Reply:
1.7.3 requires scala 2.12.13 which I'm not able to update as it does not work with scoverage